lms/djangoapps/mobile_api/users/views.py

```diff
@@ -340,6 +341,13 @@ def get_queryset(self):
         ).order_by('created').reverse()
         org = self.request.query_params.get('org', None)
 
+        try:
+            ## .. filter_implemented_name: CourseEnrollmentQuerysetRequested
+            ## .. filter_type: org.openedx.learning.course_enrollment_queryset.requested.v1
+            enrollments = CourseEnrollmentQuerysetRequested.run_filter(enrollments=enrollments)
+        except CourseEnrollmentQuerysetRequested.PreventEnrollmentQuerysetRequest as exc:
+            raise EnrollmentRequestNotAllowed(str(exc)) from exc
```

Comment:
Remember to add tests for this implementation. You could use [these tests](https://github.com/openedx/edx-platform/blob/master/common/djangoapps/student/tests/test_filters.py#L114) as guidance.

Reply:
^ I second this comment again